functions/behat.sh

```diff
@@ -121,7 +121,8 @@ function drupal_ti_ensure_chrome_driver() {
 	cd $DRUPAL_TI_BIN_DIR
 
 	# @todo Make version configurable.
-	wget http://chromedriver.storage.googleapis.com/2.13/chromedriver_linux64.zip
+	local CHROMEDRIVER_VERSION=`wget http://chromedriver.storage.googleapis.com/LATEST_RELEASE -qO-`
```

Comment:
Code style in this project does not use backticks and I don't think we need 'local', but pretty happy over the approach in general.